apple/internal/apple_framework_import.bzl

```diff
@@ -52,9 +68,23 @@ def filter_framework_imports_for_bundling(framework_imports):
 
     return filtered_imports
 
-def _apple_framework_import_impl(ctx):
-    """Implementation for the apple_framework_import rule."""
-    framework_imports = ctx.files.framework_imports
+def _all_framework_binaries(frameworks_groups):
```

Comment:
can you add a docstring, something like:
`"""Returns all the imported binary files."""`

Reply:
done